server_test.go

```diff
@@ -112,6 +115,13 @@ var (
 	context = rocket.Get("/context", func(header *rocket.Headers, cookies *rocket.Cookies) string {
 		return ""
 	})
+
+	optionalHandler = rocket.Get("/optional/", func(optionalContext *OptionalContext) string {
```

Comment:
I think named `optionalFieldHandler` is more match the functional about this handler.